apps/frontend/src/pages/Search/subcomponents/SearchFormUTR/SearchForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import s from "./SearchForm.module.scss";
+
+class SearchForm extends React.Component<
```

Comment:
Ok, du hast hier die Klassen-Deklaration von React verwendet, um die Komponente anzulegen. Das ist nicht per-se schlecht, aber wen du z.B. Hooks verwenden willst, klappt das nicht mehr so richtig. Hier macht es wahrscheinlich später Sinn, das nochmal in eine function component umzuschreiben,

Reply:
Ja, das war weil ich anfangen wollte mit fetch, state etc. zu arbeiten und diese Syntax die war die ich in Web 2 verwendet hatte, deshalb anfangs familärer, aber kann man dann auch noch umändern ab nächstem Mal ^^